rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
> expect { block }.to change { another_block } will explode (where either block has more than one line).

This is exactly what we do not want - to have `{}` blocks on more that one line. That was my impression on @lipanski intentions.

I already fixed it in Kiev gem .rubocop.yml